packages.dhall

```diff
@@ -23,3 +23,21 @@ in  upstream
     , version =
         "v5.0.1"
     }
+
+  with github-actions-toolkit =
```

Comment:
Since this is the latest package set, mind running spago upgrade-set?

Reply:
Ah! Didn't know how 😅